storage/innobase/buf/buf0flu.cc

```diff
@@ -2605,6 +2607,26 @@ buf_get_total_free_list_length(void)
 }
 
 /*********************************************************************//**
+Returns the aggregate LRU list length over all buffer pool instances.
```

Comment:
New functions should use the new InnoDB header comment style:
/** Returns ...
... */
and no /*=======*/ underneath